internal/pkg/service/templates/api/service/mapper.go

```diff
@@ -84,9 +85,19 @@ func (m Mapper) TaskPayload(model task.Task) (r *Task) {
 				InstanceID: &v,
 			}
 		}
+
+		if v, ok := model.Outputs["configId"].(string); ok {
```

Comment:
This way it's still internally a float and here it's cast to string and then parsed, isn't it?
I'd rather just use type string for it instead of uint.

Reply:
My point is that I don't see any value in typing IDs as int or uint considering they could become uuids in the future. Having IDs typed as string is more future proof imo.

Reply:
I can check with UI how they work with such values.

Reply:
Changed to `string`